dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf

```diff
@@ -54,6 +54,8 @@ server=ipv4.nsupdate.info
 protocol=dyndns2
 ssl=yes
 server=ipv6.nsupdate.info
+{%            elif account.service == 'ovh' %}
```

Comment:
you can omit the client config if it's only the protocol due to :
https://github.com/opnsense/plugins/blob/7fe1eaa4fcf74ad0fb5b3311eb04a45819e4fd78/dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf#L64-L65

Reply:
Ah right, missed that 👍🏼